modules/ROOT/pages/authentication-authorization/ldap-integration.adoc

```diff
@@ -358,6 +359,105 @@ dbms.security.ldap.authorization.access_permitted_group=501
 . Map the groups in the LDAP system to the Neo4j built-in and custom roles.
 For more information, see xref:authentication-authorization/ldap-integration.adoc#auth-ldap-map-ldap-roles[Map the LDAP groups to the Neo4j roles].
 
+[role=label--new-5.24]
+[[auth-ldap-auth-objects]]
+== Use Auth Objects to configure authentication / authorization at the user level
+xref:authentication-authorization/auth-objects.adoc[Auth Objects] can be used to determine which users can authenticate and authorize using the configured providers, including LDAP.
+
+[NOTE]
+====
+You need to change the `dbms.security.require_local_user` configuration setting to `true` to use Auth Objects. This means that a user with a matching Auth Object *must* exist in order to be able to authenticate and authorize.
+====
+
+The following examples show how you can use Cypher to do this.
+
+.Create a user with an Auth Object who can authenticate and authorize using `LDAP`.
+======
+----
+CREATE USER alice
+SET AUTH PROVIDER 'ldap' { SET ID 'cn=alice,ou=engineering,dc=example,dc=com' }
+----
+This will create a user who can authenticate and authorize using LDAP provided their LDAP `dn` is `cn=alice,ou=engineering,dc=example,dc=com`.
+
+When the `dbms.security.require_local_user` configuration setting is set to `true`, users can *only* authenticate when there is a user in the database with an Auth Object which links to the provider that the user is trying to authenticate with. If there is no matching Auth Object, the user will not be able to authenticate or authorize. This applies to all providers.
```

Comment:
```suggestion
When the `dbms.security.require_local_user` configuration setting is set to `true`, users can *only* authenticate when there is a user in the database with an Auth Object which links to the provider that the user is trying to authenticate with.
If there is no matching Auth Object, the user will not be able to authenticate or authorize. This applies to all providers.
```
just to not have that long lines (and docs prefer splitting on sentences and not having line breaks in the middle of sentences)